src/main/java/io/github/syst3ms/skriptparser/file/FileParser.java

```diff
@@ -43,7 +43,11 @@
             }
             var lineIndentation = FileUtils.getIndentationLevel(line, false);
             if (lineIndentation > expectedIndentation) { // The line is indented too much
-                logger.error("The line is indented too much (line " + (lastLine + i) + ": \"" + content + "\")", ErrorType.STRUCTURE_ERROR);
+                logger.error(
+                        "The line is indented too much (line " + (lastLine + i) + ": \"" + content + "\")",
+                        ErrorType.STRUCTURE_ERROR,
+                        "You only need to indent once (using tabs) after each section. Try to omit some tabs so the line suffices this rule"
```

Comment:
I think people don't know what a section is. Maybe explain that a section is each line with a ``:`` at the end?

Reply:
Good call.